src/replays/_replayswidget.py

```diff
@@ -501,19 +502,30 @@ def onlineTreeClicked(self, item):
                     item.generateInfoPlayersHtml()
 
     def onlineTreeDoubleClicked(self, item):
-        if hasattr(item, "duration"):
+        if hasattr(item, "duration"):  # it's a game not a date separator
             if "playing" in item.duration:  # live game will not be in vault
-                if not item.live_delay:  # live game under 5min
-                    name = None
-                    if item.mod == "ladder1v1":
-                        name = item.name[:item.name.find(" ")]  # "name vs name"
-                    else:
-                        players = (p for t in item.teams for p in t)
-                        for player in players:
-                            name = player["name"]
-                            if name != "" and name in self._playerset:
+                # search result isn't updated automatically - so game status might have changed
+                if item.uid in self._gameset.games:  # game still running
+                    game = self._gameset.games[item.uid]
+                    if not game.launched_at:  # we frown upon those
+                        return
+                    if game.has_live_replay:  # live game over 5min
+                        for name in game.players:  # find a player ...
+                            if name in self._playerset:  # still logged in
+                                self._startReplay(name)
                                 break
-                    self._startReplay(name)
+                    else:
+                        wait_str = time.strftime('%M Min %S Sec', time.gmtime(game.LIVE_REPLAY_DELAY_SECS -
+                                                                              (time.time() - game.launched_at)))
+                        QtWidgets.QMessageBox.information(client.instance, "5 Minute Live Game Delay",
+                                                          "This Game is still too young to join.\n"
+                                                          "You have to wait " + wait_str + " to join.")
+                else:  # game ended - ask to start replay
+                    if QtWidgets.QMessageBox.question(client.instance, "Live Game ended", "Want to try the Replay?",
```

Comment:
I'd go with "Would you like to watch a replay from the vault?"

Reply:
well, the user double clicked on a specific game - so the message refered to that game - hence "... the Replay" of that game the user clicked. (and "try" because replays can be corrupted, hence not starting)

Reply:
I know, I just think a bit more formal style goes better with all other strings in the client, "Want to try a replay?" is a little bit too direct IMO.

Reply:
Might be, people here are so easy offended ;-)

Reply:
It's not that, it's a matter of having an elegant UI. I deal with enough broken korglish daily, I want our client to look 100% professional in this regard :)